packages/graphql-api/src/resolvers/verify-email.ts

```diff
@@ -1,3 +1,34 @@
-export const verifyEmail = Accounts =>
-  (async (_, { token }) =>
-    await Accounts.verifyEmail(token));
+import { AccountsServer } from '@accounts/server';
+import { IResolverContext } from '../types/graphql';
+
+export const verifyEmail = (accountsServer: AccountsServer) => async (
+  _,
+  args: GQL.IVerifyEmailOnMutationArguments,
+  ctx: IResolverContext
+) => {
+  const { token } = args;
+
+  const password: any = accountsServer.getServices().password;
```

Comment:
Would be great to get ride of the `any` so we enable type checking :)

Reply:
Updated, but there is still a problem, see accounts-js/accounts#355.